src/bundle/DependencyInjection/EzPlatformRichTextExtension.php

```diff
@@ -25,6 +26,7 @@ class EzPlatformRichTextExtension extends Extension implements PrependExtensionI
     const RICHTEXT_CUSTOM_STYLES_PARAMETER = 'ezplatform.ezrichtext.custom_styles';
     const RICHTEXT_CUSTOM_TAGS_PARAMETER = 'ezplatform.ezrichtext.custom_tags';
     const RICHTEXT_ALLOY_EDITOR_PARAMETER = 'ezplatform.ezrichtext.alloy_editor';
+    public const RICHTEXT_CONFIGURATION_PROVIDER_TAG = 'ezrichtext.configuration.provider';
```

Comment:
I suppose we prefix all service tags with either `ezplatform` or `ezpublish`?

Reply:
> I suppose we prefix all service tags with either `ezplatform` or `ezpublish`?

Ok, fixed prefixed via `ezplatform` 45b1cba1eab314f997869860c8ec55a65ab815bd. We're not going to add `ezpublish` to any new code.
Also fixed remaining service tag names via #65